pkgs/applications/science/misc/openmodelica/omlibrary/update-src-libs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

/bin/sh is a symlink to bash on nixos and dash on Debian like systems which can cause problems.

Also this must be a nix-shell or most of the tool used below are not available.